src/apps/content-editor/src/app/views/ItemList/SetRow/ToggleCell/ToggleCell.js

```diff
@@ -15,26 +18,39 @@ export const ToggleCell = memo(function ToggleCell(props) {
       if (options[0].length > 3 || options[1].length > 3) {
         return (
           <span className={cx(props.className, styles.ToggleCell)}>
-            <ToggleButton
-              name={props.name}
+            <ToggleButtonGroup
+              color="secondary"
               value={props.value}
-              offValue="&nbsp;&nbsp;&nbsp;"
-              onValue="&nbsp;&nbsp;&nbsp;"
-              onChange={props.onChange}
-            />
-            {props.field.settings.options[props.value]}
+              size="small"
+              onChange={(e, val) => {
+                if (val !== null) {
```

Comment:
Why are we doing a `null` check? Shouldn't the possible values be limited to the defined `ToggleButton` props of `1` and `2`?

Reply:
Null is returned when clicking an already active value. Mui does this to provide us a way to not trigger unnecessary actions occur when clicking an already active option

Reply:
I think this guard adds more conceptual overhead and code than it is worth. The likelihood of clicking the same toggle selection seems low and the overhead of emitting that click seems low vs the understanding needed every time this code is revisited of "why are we checking null?"

@agalin920 thoughts?

Reply:
@shrunyan Just want to note that for some of the actions that take place within this callback can potentially break the app if active button is clicked and callback passes null to an action. Should we

1. Not do a null check on those and be aware that this issue ca occur
2. Leave the null check as is
3. Move null check to action as opposed to callback

Reply:
Lets implement 3. So we can write once. Also lets ensure we have a thorough comment explaining why and how it relates to MUI

Reply:
@d88naimi please implement the solution as described by @agalin920. Connect with him if that needs clarification.